pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -205,8 +213,9 @@ func toLBProtocol(protocol corev1.Protocol) string {
 	}
 }
 
-func (lbaas *CloudLb) createLoadBalancer(service *corev1.Service, name string, clusterName string, internalAnnotation bool, port corev1.ServicePort) (*loadbalancers.LoadBalancer, error) {
-
+func createLBCreateVips(service *corev1.Service) ([]virtualips.CreateOpts, error) {
+	//internalAnnotation, err := getBoolFromServiceAnnotation(apiService, ServiceAnnotationLoadBalancerInternal, false)
+	internalAnnotation := false
 	var lbType string
 	switch internalAnnotation {
```

Comment:
switch isn't needed here after setting internalAnnotation to false.  
possible to change internalAnnotation and lbType to constants as well.  performance gain from doing so is almost non-existent (0.0036 nanoseconds per op gain) but might read to less confusion for future coder, or future you.

Reply:
ah that was part of a planned switch to reading the value from the service annotation that hasn't happened yet

Reply:
I'll leave it for a reminder to uncomment the above code.